library/psa_crypto.c

```diff
@@ -3592,6 +3592,7 @@ static psa_status_t psa_aead_check_nonce_length( psa_algorithm_t alg,
             break;
 #endif /* PSA_WANT_ALG_CHACHA20_POLY1305 */
         default:
+            (void) nonce_length;
```

Comment:
Should we guard this cast with `#if !GCM && !CCM && !ChachaPoly` or is it overkill?

Reply:
Hmm it doesn't affect the binary size (just silent compiler), so it looks ok for me.